tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/bigquery_tests_base.py

```diff
@@ -95,6 +95,7 @@ def setUpClass(cls):
                 cls.client.create_dataset(dataset, exists_ok=True)
 
         def setUp(self):
+            self.tearDownClass()
```

Comment:
Should this still be here?

Reply:
I think this will help with testing regardless of what the initial state of the datasets are, left over from previous tests or re-running the tests.

Reply:
Oh, okay. Gotcha!